tailwind.config.js

```diff
@@ -1,8 +1,11 @@
 module.exports = {
-  purge: ['./src/*.js', './src/components/**/*.js'],
-  theme: {
-    extend: {},
-  },
-  variants: {},
-  plugins: [],
+	purge: ['./src/*.js', './src/components/**/*.js'],
+	theme: {
+		extend: {},
+		height: {
```

Comment:
And now you no longer need to change the config. Just remove this commit and I'll be ready to merge your commit change :)